code/modules/hydroponics/grown/potato.dm

```diff
@@ -33,7 +33,17 @@
 			user << "<span class='notice'>You add some cable to the potato and slide it inside the battery encasing.</span>"
 			var/obj/item/weapon/stock_parts/cell/potato/pocell = new /obj/item/weapon/stock_parts/cell/potato(user.loc)
 			pocell.maxcharge = seed.potency * 20
+
+			// The secret of potato supercells!
+			var/datum/plant_gene/trait/cell_charge/G = seed.get_gene(/datum/plant_gene/trait/cell_charge)
```

Comment:
This needs to be both generic as in glow in new and you need to move it up the level of inheritance if you want all the plants be able to be made into cells
Also a system that makes a sprite for the plant cell together with description